spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -87,17 +90,18 @@
 
 	@Test
 	public void testAutoCommit() throws Exception {
-		logger.info("Start auto");
+		this.logger.info("Start auto");
 		Map<String, Object> props = KafkaTestUtils.consumerProps("test1", "true", embeddedKafka);
 		DefaultKafkaConsumerFactory<Integer, String> cf = new DefaultKafkaConsumerFactory<Integer, String>(props);
 		ConcurrentMessageListenerContainer<Integer, String> container =
 				new ConcurrentMessageListenerContainer<>(cf, topic1);
 		final CountDownLatch latch = new CountDownLatch(4);
+		final ConcurrentSkipListSet<String> listenerThreadNames = new ConcurrentSkipListSet<>();
```

Comment:
`ConcurrentSkipListSet` just for `CountDownLatch(4)`? :smile: 
Minor of course, but really it is curious how that was selected.
Thanks